web/styles/informational_overlays.css

```diff
@@ -48,6 +48,20 @@
         }
     }
 
+    .overlay-body {
+        table {
+            border-color: var(--color-border-informational-overlays-table);
+        }
+
+        & tr th {
+            border-color: var(--color-border-informational-overlays-table);
+        }
+
+        & tr td {
+            border-color: var(--color-border-informational-overlays-table);
+        }
+    }
```

Comment:
I can merge this to resolve the regression, but it'd better to assign these styles to actual classes that we define in those tables.